src/store/actions/actionCreators.js

```diff
@@ -0,0 +1,20 @@
+import { SIGNUP } from './types';
+
+export const signUp = user => async dispatch => {
+  await fetch('https://authors-haven-odin.herokuapp.com/api/users/', {
```

Comment:
You are using API hard-coded URLs here  which is fine, but think of how many places you would need to change if our domain for the API changed, say we bought a real domain like `www.authorhavens.com`, luckily we already have a helper in `src/helpers/constants` that mitigates this sort of issue.